Makefile

```diff
@@ -1,11 +1,23 @@
-COMP := target/debug/sylt
+SYLT := target/debug/sylt
 
-.PHONY: clean
+.PHONY: all clean clean-lua $(SYLT) run-main
 
-%.lua: %.sy $(COMP)
-	@$(COMP) --lua $<
-	@echo "------ RUN ------"
-	@lua $@
+all: $(SYLT)
 
-clean:
-	rm -rf *.lua
+run-main: main.love
+	love .
+
+%.love: %.lua
+	cp $< $@
```

Comment:
This should be a zip.
Something like this:
```
zip $< *.lua
```
But probably more.

I don't think you mean to have this rule here.

Reply:
I don't think we depend on `*.lua` since all relevant .sy-files are compiled into the same file.